workspace.spec

```diff
@@ -572,6 +572,359 @@ module Workspace {
 		string description;
 	} ProvenanceAction;
 
+	/* Credit Metadata-related objects and fields
+
+		Workspace objects have a list of zero or more sets of credit metadata (CM), or citation information for the object. This information allows data imported into KBase to retain the appropriate details for users wishing to cite the data source, and for those who contributed to its creation to be credited for their work.
+
+		CM is stored in a list as part of the object provenance, but unlike other provenance information, which is immutable, CM can be updated.
+
+		Credit metadata is added after object creation, and requires the user to be a workspace admin. Username, timestamp, and credit metadata schema version are stored along with the CM, which allows tracking of CM changes.
+
+		To update existing CM, a new set of credit metadata is added to the existing list.
+
+		When requesting credit metadata, the last (i.e. most recent) entry in the list is returned.
+
+		In the following entries, "resource" is used to refer to the workspace object that the CM pertains to.
+	*/
+
+	/* EventDate
+
+		Represents an event in the lifecycle of a resource and the date it occurred on.
+		These objects are used to capture information about
+
+		Both event and date are required fields.
+
+		string date - the date associated with the event. The date may be in the format
+			YYYY, YYYY-MM, or YYYY-MM-DD.
+		string event - the event that occurred; events must be from the following list:
+			"accepted"
+			"available"
+			"copyrighted"
+			"collected"
+			"created"
+			"issued"
+			"submitted"
+			"updated"
+			"valid"
+			"withdrawn"
+			"other"
+	*/
+	typedef structure {
+		string date;
+		string event;
+	} EventDate;
+
+	/* FundingReference
+
+		Represents a funding source for a resource, including the funding body and the grant awarded.
+
+		The 'funder_name' field is required; all others are optional.
+
+		string funder_id (optional) - persistent unique identifier for the funder in the format
+			<database name>:<identifier within database>
+		string funder_name - the funder name
+		string award_id (optional) - code for the award, assigned by the funder
+		string award_title (optional) - title for the award
+		string award_url (optional) - URL for the award
+	*/
+	typedef structure {
+		string funder_id;
+		string funder_name;
+		string award_id;
+		string award_title;
+		string award_url;
+	} FundingReference;
+
+	/* Organization
+
+		Represents an organization.
+
+		'organization_name' is required; 'organization_id' is optional.
```

Comment:
Can we recommend a place to look for these? e.g., ror.org/
Also, if there is somewhere that has "standards" for Org Names, that would useful 🤦

Reply:
ror.org is probably as good a place as any to look for organization names -- I'll add that to the documentation